boards.txt

```diff
@@ -28,6 +28,9 @@ arduino_101.build.variant_system_lib=arc32drv_arduino101
 arduino_101.build.vid=0x8087
 arduino_101.build.pid=0x0AB6
 
+arduino_101.bootloader.tool=arduino101load
+arduino_101.bootloader.file=""
```

Comment:
I saw that too. Since that file isn't used, perhaps we can point that to the actual `image.bin` to suppress the error?